README.md

```diff
@@ -410,6 +410,42 @@ with (_) {
 trace(_.sort(bar), _.trim(baz));
 ```
 
+### Use shorthand object properties
+> Reason: it makes code cleaner
+```javascript
+// BAD
+const foo = {
+    a: a,
+    b: b,
+    c: function() {},
+};
+
+// GOOD
+const foo = {
+    a, 
+    b,
+    c() {},
+};
+
+```
+Move shorthand properties up
```

Comment:
I guess it would be better to make it as a separate paragraph. It could be shared via separate link. What do you think?

Reply:
Yes, good point, but I think the separate paragraph is too much, I have just added a link.

Reply:
ok, it's also good.